interactive_demo.py

```diff
@@ -64,32 +71,43 @@
             help='Resize the shorter side to this size. -1 to use original resolution. ')
     args = parser.parse_args()
 
+    # create temporary workspace if not specified
     config = vars(args)
     config['enable_long_term'] = True
     config['enable_long_term_count_usage'] = True
 
-    with torch.cuda.amp.autocast(enabled=not args.no_amp):
+    if config["workspace"] is None:
```

Comment:
ResourceManager should have handled this. Am I missing something?

Reply:
Oh you are correct, I simply didn't notice that the ResourceManager kept a workspace variable. I will modify the PR to reflect these changes.